cmd/litcli/ln.go

```diff
@@ -297,41 +297,46 @@ var sendPaymentCommand = cli.Command{
 	Action: sendPayment,
 }
 
-func sendPayment(ctx *cli.Context) error {
+func sendPayment(cliCtx *cli.Context) error {
```

Comment:
nit: `cli` is maybe not an optimal parameter name for the pattern if we shadow the `cli` package (in the other commands)

Reply:
yeah tricky. But opted to prio having `ctx` be used for `context.Context` since that is standard.